Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/impl/workspace/TextFileFactory.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.spring.quadriga.domain.factory.impl.workspace;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.factory.workspace.ITextFileFactory;
+import edu.asu.spring.quadriga.domain.impl.workspace.TextFile;
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+
+/**
+ * @author Nischal Samji Implementation class for TextFile Interface. Creates
```

Comment:
first comes the description, then the author info, etc
http://www.oracle.com/technetwork/articles/java/index-137868.html